root-packages/v4l-utils/build.sh

```diff
@@ -0,0 +1,39 @@
+# http(s) link to package home page.
+TERMUX_PKG_HOMEPAGE=https://linuxtv.org
+
+# One-line, short package description.
+TERMUX_PKG_DESCRIPTION="Userspace tools and conversion library for Video 4 Linux"
+
+# License.
+TERMUX_PKG_LICENSE="GPL-2.0-or-later"
```

Comment:
Please change to `GPL-2.0`

Reply:
It's kind of a mish-mash, some files are 2.0 only, some are 2.1 only, some are 2.0+.